client/site-profiler/components/site-profiler-v2.tsx

```diff
@@ -162,6 +162,7 @@ export default function SiteProfilerV2( props: Props ) {
 						<ResultsHeader
 							domain={ domain }
 							performanceCategory={ performanceCategory }
+							isWordPress={ ! noWordPressFound }
```

Comment:
Is this the best way to define if a site is made with WordPress or not? As this value comes from the security request errors I'm wondering if there is a different way to get this value.